src/pydvl/utils/numeric.py

```diff
@@ -96,7 +96,7 @@ def random_subset(
         The subset
     """
     rng = np.random.default_rng(seed)
-    selection = rng.uniform(size=len(s)) > q
```

Comment:
I guess this is the bug fix, correct? So you could add "Fix a bug in ..., due to incorrect comparison operator"

Reply:
Agree, done.